### PR TITLE
Add built-in tuner test in CI

### PR DESCRIPTION
**Pull request by Crysple**
- Check whether each built-in tuner can be run normally in travis CI

### CODE REVIEW

File: test/test_builtin_tuner/naive_trial.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import nni
+
+params = nni.get_parameters()
+print('params:', params)
```
Comment (xuehui1991) [inline test/test_builtin_tuner/naive_trial.py]: shall we use print as test method?? Or use unit test to instead?

File: test/test_builtin_tuner/run.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
```
Comment (xuehui1991) [inline test/test_builtin_tuner/run.py]: I'm not sure shall we add license here....@quanlu zhang

File: test/test_builtin_tuner/search_space.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (xuehui1991) [inline test/test_builtin_tuner/search_space.json]: shall we add all kind of search space to test their format?

**Comment by Crysple** [comment]
This pr is not ready. Further modification needs to be done because of refracting exp stop #113

File: test/test_builtin_tuner/run.py (hunk 2 above)
Comment (QuanluZhang) [reply test/test_builtin_tuner/run.py]: yes, could add license here

**Comment by QuanluZhang** [comment]
please refactor this pr, since we have new experiment stopping logic.

**Comment by Crysple** [comment]
Will refactor this pr

**Pull request closed without merging**